functions/ingest/format/defaulttsdb.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/v3io/v3io-tsdb/pkg/tsdb"
 	"github.com/v3io/v3io-tsdb/pkg/utils"
+	"strings"
```

Comment:
dont know how lint passed for you, but i the standard is to put all "golang" imports before other dependencies..